app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, (t ".title")) %>
```

Comment:
1. em có thể bỏ ngoặc của `provide` được nhé
ngoài ra `(t ".title")` thì em nên sửa thành `t(".title")` nhé
 => `<% provide :title, t(".title") %>`
2. Dùng I18n cho toàn bộ file chứ ko mỗi phần title mô (khoc)
self check và fix lại giúp chị phát nhé
Note thêm, dùng link_to thay cho tag `a` nhé